packages/esm-patient-medications-app/translations/en.json

```diff
@@ -40,6 +40,7 @@
   "patientInstructions": "Patient Instructions",
   "patientInstructionsPlaceholder": "Additional dosing instructions (e.g. \"Take after eating\")",
   "pillsDispensed": "Pills dispensed",
+  "pillstoDispense": "Pills to dispense"
```

Comment:
Needs a comma after the end of line 43 @gracepotma

Reply:
```suggestion
  "pillsToDispense": "Pills to dispense",
```

Reply:
Oops. Thanks for this catch @denniskigen 😅 .